spec/outputs/elasticsearch_spec.rb

```diff
@@ -306,6 +306,56 @@
     end
   end
 
+  describe "http client create" do
+    require "logstash/outputs/elasticsearch"
+    require "elasticsearch"
+
+    let(:options) {
+      {
+        "protocol" => "http",
+        "index" => "my-index",
+        "host" => "localhost",
+        "path" => "some-path"
+      }
+    }
+
+    let(:eso) {LogStash::Outputs::ElasticSearch.new(options)}
+
+    let(:manticore_host) {
+      eso.client.first.send(:client).transport.options[:host].first
+    }
+
+    around(:each) do |block|
+      thread = eso.register
```

Comment:
K! I just noticed that it happened to return a thread. It made sense to me to just go ahead and kill it. I'm not sure if that's necessary though. Do you have any ideas there? I assumed it was intended to either be join()ed or kill()ed. I'll look into why this is returning a thread.